src/ontology/oba.Makefile

```diff
@@ -139,14 +160,24 @@ $(REPORTDIR)/%.tsv: ../sparql/synonyms-exact.sparql $(TMPDIR)/%.owl
 prepare_oba_alignment: $(REPORTDIR)/uberon.tsv $(REPORTDIR)/efo.tsv $(REPORTDIR)/pato.tsv $(REPORTDIR)/oba.tsv $(REPORTDIR)/vt.tsv $(REPORTDIR)/cl.tsv $(REPORTDIR)/go.tsv $(REPORTDIR)/chebi.tsv
 	echo "OK cool all tables prepared."
 
-CHECK_SPARQL=oba.owl
+#This is the OBA-VT DIFF Pipeline
+
+$(TMPDIR)/mirror-%.owl:
+	wget $(OBOBASE)/$*.owl -O $@
+
+
+$(TMPDIR)/lexmatch-%.sssom.tsv: $(TMPDIR)/%.db
+	runoak -i $< lexmatch -o $@
+
+$(TMPDIR)/merge-oba-vt.owl: $(TMPDIR)/mirror-oba.owl $(TMPDIR)/mirror-vt.owl
+	$(ROBOT) merge -i $< -i $(TMPDIR)/mirror-vt.owl -o $@
 
-check_children_oba: $(CHECK_SPARQL)
-	$(ROBOT) verify -i $< --queries ../sparql/biological-attribute-child-violation.sparql -O $(REPORTDIR)
+$(TMPDIR)/merge-oba-vt.db: $(TMPDIR)/merge-oba-vt.owl 
+	semsql make $@
 
-test: check_children_oba
+$(REPORTDIR)/oba-vt-diff.yaml: $(TMPDIR)/merge-oba-vt.db
+	runoak --stacktrace diff-via-mappings --mapping-input ../mappings/oba-vt.sssom.tsv -X $< -o $@
```

Comment:
This currently fails with: 

`runoak --stacktrace diff-via-mappings --mapping-input ../mappings/oba-vt.sssom.tsv -X merge-oba-vt.db -o oba-vt-diff.yaml
WARNING:root:No prefix map provided (not recommended), trying to use defaults..
NotImplementedError
make: *** [oba.Makefile:186: oba-vt-diff.yaml] Error 1`

Reply:
@hrshdhgd Do you know what to do?

Reply:
@souzadevinicius also, once you figured this out, please make sure the OAK docs are updated with a better more clear example so that future generations do not puzzle over this!

Reply:
Guys, thanks for all your suggestions, but none of them works.

Below are the detailed commands and their results

## @matentzn  approach
```
runoak --stacktrace diff-via-mappings --mapping-input ../mappings/oba-vt.sssom.tsv -i sqlite:tmpoba.db -o oba-vt-diff.yaml
Usage: runoak diff-via-mappings [OPTIONS] [TERMS]...
Try 'runoak diff-via-mappings --help' for help.
Error: No such option: -i
```
---

## @hrshdhgd  approach
### Using merged input
```
runoak --stacktrace diff-via-mappings --mapping-input ../mappings/oba-vt.sssom.tsv -X sqlite:tmp/merge-oba-vt.db -o oba-vt-diff.yaml
WARNING:root:No prefix map provided (not recommended), trying to use defaults..
NotImplementedError
```
---
### Using two inputs
```
runoak --stacktrace diff-via-mappings --mapping-input ../mappings/oba-vt.sssom.tsv -X sqlite:tmp/mirror-oba.db -X sqlite:tmp/mirror-vt.db -o oba-vt-diff.yaml
WARNING:root:No prefix map provided (not recommended), trying to use defaults..
NotImplementedError
```